### ISSUE TITLE
Loading of Inverted icons using read_viz_icons

### ISSUE THREAD

**Issue by Nibba2018** [fury-gl/fury#199]
![image](https://user-images.githubusercontent.com/29832615/76645645-6b14d200-657f-11ea-9860-4a7e3040aeee.png)

Icons loaded by read_viz_icons are inverted when displayed.

### Code snippet:

```python
from fury.data import read_viz_icons
from fury import ui, window
button_example = ui.Button2D(icon_fnames=[('Up', read_viz_icons(fname='circle-up.png'))], position=(50,50), size=(30,30))
current_size = (100, 100)
show_manager = window.ShowManager(size=current_size, title="Inverted Image BUG")
show_manager.scene.add(button_example)
show_manager.start()
```

**Comment by skoudoro** [comment]
Thank you for this feedback @Nibba2018!

we will look into it. we need to update `read_viz_icons` and add a test. it should have failed.

**Comment by Nibba2018** [comment]
Is this somehow related to the bug which inverted images while saving?

**Comment by skoudoro** [comment]
It is fixed on my PR #201
